src/dotnet/APIView/APIViewWeb/Controllers/AccountController.cs

```diff
@@ -12,6 +12,10 @@ public class AccountController : Controller
         public async Task<IActionResult> Login(string returnUrl = "/")
         {
             await HttpContext.SignOutAsync();
+            if (!Url.IsLocalUrl(returnUrl))
```

Comment:
Interesting check

Reply:
Since we are now including the returnUrl in the Login link, though I don't think this can ever actually be set to an external Url without the user modifying it after getting directed to the unauthorized page.